packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -778,6 +786,10 @@ export class GarbageCollector implements IGarbageCollector {
         this.updateCurrentState(gcData, gcResult, currentReferenceTimestampMs);
         this.runtime.updateUsedRoutes(gcResult.referencedNodeIds, currentReferenceTimestampMs);
 
+        // Log events for objects that are ready to be deleted by sweep. When we have sweep enabled, we will
+        // delete these objects here instead.
+        this.logSweepEvents(logger, currentReferenceTimestampMs);
```

Comment:
Now that I think about it, if we're in sweep mode and we're in test mode, that would create some weird behavior right?

Reply:
Probably. If test mode is on, unreferenced nodes are immediately deleted so they won't get to inactive / sweep ready state. This function essentially becomes a no-op.
It's similar to inactive events which will be no-op.